docker build error

from a custom image build script, but it really doesn't do anything other than tag the image with commit. was able to reproduce this on another machine as well
```
+ git rev-parse --short HEAD
+ cmt=22f9ffa
+ docker build -t konbini:why-22f9ffa -f Dockerfile .
[+] Building 17.3s (15/16)                                                                                                                                                                                                                                                                                      docker:default
 => [internal] load build definition from Dockerfile                                                                                                                                                                                                                                                                      0.0s
 => => transferring dockerfile: 544B                                                                                                                                                                                                                                                                                      0.0s
 => [internal] load metadata for docker.io/library/alpine:latest                                                                                                                                                                                                                                                          1.4s
 => [internal] load metadata for docker.io/library/golang:1.25-alpine                                                                                                                                                                                                                                                     1.3s
 => [internal] load .dockerignore                                                                                                                                                                                                                                                                                         0.0s
 => => transferring context: 2B                                                                                                                                                                                                                                                                                           0.0s
 => [builder 1/7] FROM docker.io/library/golang:1.25-alpine@sha256:b6ed3fd0452c0e9bcdef5597f29cc1418f61672e9d3a2f55bf02e7222c014abd                                                                                                                                                                                       0.0s
 => [stage-1 1/4] FROM docker.io/library/alpine:latest@sha256:4bcff63911fcb4448bd4fdacec207030997caf25e9bea4045fa6c8c44de311d1                                                                                                                                                                                            0.0s
 => [internal] load build context                                                                                                                                                                                                                                                                                         0.0s
 => => transferring context: 47.69kB                                                                                                                                                                                                                                                                                      0.0s
 => CACHED [stage-1 2/4] RUN apk --no-cache add ca-certificates                                                                                                                                                                                                                                                           0.0s
 => CACHED [stage-1 3/4] WORKDIR /root/                                                                                                                                                                                                                                                                                   0.0s
 => CACHED [builder 2/7] WORKDIR /app                                                                                                                                                                                                                                                                                     0.0s
 => CACHED [builder 3/7] RUN apk add --no-cache git                                                                                                                                                                                                                                                                       0.0s
 => [builder 4/7] COPY go.mod go.sum ./                                                                                                                                                                                                                                                                                   0.0s
 => [builder 5/7] RUN go mod download                                                                                                                                                                                                                                                                                    15.1s
 => [builder 6/7] COPY *.go ./                                                                                                                                                                                                                                                                                            0.1s
 => ERROR [builder 7/7] RUN CGO_ENABLED=0 GOOS=linux go build -o konbini .                                                                                                                                                                                                                                                0.5s
------
 > [builder 7/7] RUN CGO_ENABLED=0 GOOS=linux go build -o konbini .:
0.454 main.go:31:2: no required module provides package github.com/whyrusleeping/konbini/xrpc; to add it:
0.454   go get github.com/whyrusleeping/konbini/xrpc
------
Dockerfile:17
--------------------
  15 |
  16 |     # Build the application
  17 | >>> RUN CGO_ENABLED=0 GOOS=linux go build -o konbini .
  18 |
  19 |     # Runtime stage
--------------------
ERROR: failed to build: failed to solve: process "/bin/sh -c CGO_ENABLED=0 GOOS=linux go build -o konbini ." did not complete successfully: exit code: 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,7 +11,7 @@ COPY go.mod go.sum ./
 RUN go mod download
 
 # Copy source code
-COPY *.go ./
+COPY . .
 
 # Build the application
 RUN CGO_ENABLED=0 GOOS=linux go build -o konbini .

